### ISSUE TITLE
"Visible token storage is deprecated and will be removed in the next release" warning

### ISSUE THREAD

**Issue by andypiper** [vscode-glitch/vscode-glitch#22]
Just a quick note here that the `glitch.token` line in my settings.json is now generating a warning from VSCode (per the title). Might be worth looking back at #6 in case this breaks in future.

**Comment by dluxemburg** [comment]
Yes this is intentional (though seems to have caused more trouble than intended). New sign-ins will store the token using an integration with the system keychain so that is cannot be easily stolen.
